.github/workflows/release.yml

```diff
@@ -59,8 +59,12 @@ jobs:
           ./build/bin/nvim --version | head -n 3 >> $GITHUB_OUTPUT
           printf 'END\n' >> $GITHUB_OUTPUT
 
-  macOS:
-    runs-on: macos-14
+  macos:
```

Comment:
**nit:** Why that rename? `macOS` is the correct spelling.

Reply:
I dislike mixing upper case and lower case in code.

Reply:
As long as output remains correct, that is fine.